### PR TITLE
First draft of ESLint settings and package infrastructure.

### PR DESCRIPTION
**Pull request by dylankerr-bis**

### CODE REVIEW

**Review by dougstowell — APPROVED** [review]
Looks good.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "eslint-config-bis",
+  "version": "0.0.1",
+  "description": "ESLint configuration to help comply with BIS JavaScript standards.",
+  "main": "index.js",
+  "author": "Dylan Kerr",
+  "licence": "ISC",
```
Comment (benbroadley-bis) [inline package.json:7]: I take it that this is the correct licence that we want to use.

**Review by benbroadley-bis — APPROVED** [review]
Just one comment on the licence, I take it it's the correct one to use?

File: package.json (hunk 1 above)
Comment (dylankerr-bis) [reply package.json:7]: Confirmed with Tom.

**Pull request merged**